src/components/addProductsForm/AddProducts.jsx

```diff
@@ -1,5 +1,177 @@
-import React from "react";
+import React, { useState } from "react";
+import { Form, Button, Row, Col, Container } from "react-bootstrap";
+import "./style.scss";
+import db from "../config/firebaseConfig";
 
 export default function AddProducts() {
-  return <div>add Products</div>;
+  const [hasSizes, setHasSizes] = useState(false);
+  const [productFormBrand, setProductFormBrand] = useState("");
+  const [productFormTitle, setProductFormTitle] = useState("");
+  const [productFormDescription, setProductFormDescription] = useState("");
+  const [productFormCategory, setProductFormCategory] = useState("");
+  const [productFormPrice, setProductFormPrice] = useState("");
+
+  // Add products to the database
+  const addNewProductToDatabase = async (e) => {
+    e.preventDefault();
+    // Getting all images and push them into an array
+    const allImages = [];
+    const arrayOfImages = [
```

Comment:
this code really confuses me. for example, i read this before, but now i have to figure out

from an outside point of view, what are firstImage, secondImage, thirdImage, etc. why are there magic strings for things? are they IDs? are they translation keys? are they special values for the db? its completely unknown to the reader at the moment.

same with firstSize etc

variable names and comments need to clarify this.